CONTRIBUTING.md

```diff
@@ -114,11 +114,74 @@ Rustup is distributed. The steps for a release are:
 
 ## Developer tips and tricks
 
-The environment variable RUSTUP_FORCE_ARG0 can be used to get rustup to think
+### `RUSTUP_FORCE_ARG0`
+
+The environment variable `RUSTUP_FORCE_ARG0` can be used to get rustup to think
 its a particular binary, rather than e.g. copying it, symlinking it or other
 tricks with exec. This is handy when testing particular code paths from cargo
 run.
 
 ```
 $ RUSTUP_FORCE_ARG0=rustup cargo run -- uninstall nightly
 ```
+
+### `RUSTUP_BACKTRACE`
+
+By default while running tests, we unset some environment variables that will
+break our testing (like `RUSTUP_TOOLCHAIN`, `SHELL`, `ZDOTDIR`, `RUST_BACKTRACE`).
+But if you want to debug locally, you may need backtrace. The `RUSTUP_BACKTRACE`
+is used like `RUST_BACKTRACE` to enable backtrace of failed test.
```

Comment:
`to enable backtraces of failed tests.`

Reply:
Maybe worth adding: `**NOTE**: This is a backtrace for the test, not for any rustup process running in the test`